src/repo/zcl_abapgit_dot_abapgit.clas.abap

```diff
@@ -263,6 +238,30 @@ CLASS ZCL_ABAPGIT_DOT_ABAPGIT IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD get_translation_languages.
+
+    " Returns a list of translation languages for serialization
+    " If the setting is initial, no translations shall be serialized
+    " If the setting is `*`, all all installed system languages shall be serialized
+    " Else, the setting shall contain all languages to be serialized
+
+    DATA lv_installed_languages TYPE string.
+
+    IF ms_data-i18n_languages IS NOT INITIAL.
+      READ TABLE ms_data-i18n_languages TRANSPORTING NO FIELDS WITH KEY table_line = '*'.
+      IF sy-subrc = 0.
+        rt_languages = zcl_abapgit_lxe_texts=>get_installed_languages( ).
```

Comment:
mmm, tight coupling ... I'd not mix it inside dotabapgit, it should be elsewhere imho. Somewhere in zcl_abapgit_lxe_texts class

Reply:
done